cmd/common/containerized-gadgets/push.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package containerizedgadgets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"oras.land/oras-go/v2"
+	"oras.land/oras-go/v2/registry/remote"
+)
+
+type pushOptions struct {
+	tag string
+}
+
+func NewPushCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "push tag",
```

Comment:
nit: `tag` -> `TAG` to make it clear that it's an argument. (Same in other commands).